moped-editor/src/views/projects/projectView/ToolbarPlugin.js

```diff
@@ -0,0 +1,287 @@
+import { useCallback, useEffect, useState } from "react";
+
+import {
+  Box,
+  ButtonGroup,
+  Button,
+} from "@mui/material";
+import {
+  Redo,
+  Undo,
```

Comment:
fwiw, i would not miss the undo/redo buttons if they weren't there, assuming the built-in browser/OS equivalent is working normally.

Reply:
i think they're pretty snazzy but if anyone else feels strongly about removing them we can consider it

Reply:
I think that John raises a good point about the keyboard shortcuts handling the functionality of these buttons. If we had a mobile user base, the buttons could provide more of an important role in the UI.

In light of the [ETL issues we faced this week](https://austininnovation.slack.com/archives/CHZE6BC6L/p1727122151675579), I think that we should dedicate more code to getting plain text out of this editor in future-scoped work, and keep things simple when we roll this out.

@tillyw Does removing the undo/redo buttons feel doable?

Reply:
went ahead and removed them!